app/views/routeDetails/routing/CountryOfRoutingView.scala.html

```diff
@@ -0,0 +1,58 @@
+@*
+ * Copyright 2022 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *@
+
+@import controllers.routeDetails.routing.routes
+@import models.LocalReferenceNumber
+@import models.reference.Country
+@import models.Selectable
+@import models.Selectable._
+@import uk.gov.hmrc.govukfrontend.views.html.components.implicits._
+@import viewModels.components.InputSelectViewModel._
+@import views.html.components.{Button, ErrorSummary, InputSelect}
+
+@this(
+    mainTemplate: MainTemplate,
+    formHelper: FormWithCSRF,
+    inputSelect: InputSelect,
+    errorSummary: ErrorSummary,
+    button: Button
+)
+
+@(form: Form[Country], lrn: LocalReferenceNumber, countries: Seq[Country], mode: Mode, index: Index)(implicit request: Request[_], messages: Messages)
+
+@mainTemplate(
+    title = messages("routeDetails.routing.countryOfRouting.title"),
+    showBackLink = true,
+    lrn = Some(lrn)
+) {
+
+    @formHelper(action = routes.CountryOfRoutingController.onSubmit(lrn, mode, index), 'autoComplete -> "off") {
+
+        @errorSummary(form.errors)
+
+        @inputSelect(
+            field = form("value"),
+            selectType = OrdinarySelect(
+                heading = messages("routeDetails.routing.countryOfRouting.heading"),
+                caption = Some(messages("routeDetails.caption"))
+            ),
+            selectItems = countries.toSelectItems(form.value),
+            placeholder = messages("routeDetails.routing.countryOfRouting.placeholder")
+        )
+
+        @button(attributes = Map("id" -> "submit"))
+    }
+}
```

Comment:
Hint text missing - also from spec. It's not shown on the screenshot attached to the ticket but it is in the proposed content